Examples/MAX32655/I2C/main.c

```diff
@@ -64,7 +64,7 @@
 #define I2C_SLAVE_ADDR (0x51)
 #define I2C_BYTES 100
 
-typedef enum { FAILED, PASSED } test_t;
+typedef enum { FAILED = -1, PASSED = 0 } test_t;
```

Comment:
My suggestions are similar to what is done for this example here, except that we should move the functionality to `mxc_errors.h`.